lib/content-services/src/lib/content-metadata/components/content-metadata-card/content-metadata-card.component.spec.ts

```diff
@@ -211,14 +162,6 @@ describe('ContentMetadataCardComponent', () => {
         expect(getToggleEditButton()).toBeNull();
     });
 
-    it('should show the edit button if node does has `update` permissions', () => {
```

Comment:
what is the new behaviour? are you controlling the editing if "update" permission is not provided?